legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabWorkspaceApi.java

```diff
@@ -507,7 +507,7 @@ public WorkspaceUpdateReport updateWorkspace(String projectId, String workspaceI
         Branch masterBranch;
         try
         {
-            masterBranch = withRetries(() -> repositoryApi.getBranch(gitLabProjectId.getGitLabId(), MASTER_BRANCH));
+            masterBranch = withRetries(() -> repositoryApi.getBranch(gitLabProjectId.getGitLabId(), getDefaultBranch(gitLabProjectId)));
```

Comment:
Call getDefaultBranch outside withRetries. Also, let's rename the variable "masterBranch" to "defaultBranch".